common/src/main/scala/com/gu/i18n/CountryGroup.scala

```diff
@@ -0,0 +1,315 @@
+
+package com.gu.i18n
+
+case class CountryGroup(name: String,
+                        id: String,
+                        defaultCountry: Option[Country],
+                        countries: List[Country],
+                        currency: Currency,
+                        postalCode: PostalCode
+                       )
+
+object CountryGroup {
+
+  import Currency._
+
+  val C = Country
+  val Canada = CountryGroup("Canada", "ca", Some(C.Canada), List(C.Canada), CAD, ZipCode)
+
+  val US = CountryGroup("United States", "us", Some(C.US), List(C.US), USD, ZipCode)
+
+  val UK = CountryGroup("United Kingdom", "uk", Some(C.UK), List(
+    C.UK,
+    Country("FK", "Falkland Islands"),
+    Country("GI", "Gibraltar"),
+    Country("GG", "Guernsey"),
+    Country("IM", "Isle of Man"),
+    Country("JE", "Jersey"),
+    Country("SH", "Saint Helena")
+  ), GBP, PostCode)
+
+  val Australia = CountryGroup("Australia", "au", Some(C.Australia), List(
+    C.Australia,
+    Country("KI", "Kiribati"),
+    Country("NR", "Nauru"),
+    Country("NF", "Norfolk Island"),
+    Country("TV", "Tuvalu")
+  ), AUD, PostCode)
+
+  val NewZealand = CountryGroup("New Zealand", "nz", Some(C.NewZealand), List(
+    C.NewZealand,
+    Country("CK", "Cook Islands")
+  ), NZD, PostCode)
```

Comment:
Why do we still treat New Zealand differently from RoW? If we were to put it under `INT` would it make Zuora/Salesforce unhappy?

@rtyley @paulbrown1982